.github/workflows/test.yml

```
@@ -91,6 +91,13 @@ jobs:
         # Custom app-config.local.yaml
         cp -vr configs/app-config/app-config.local.example.yaml configs/app-config/app-config.local.yaml
 
+    - name: Create dynamic plugins directory
+      if: ${{ matrix.composeConfig.name == "dynamic-plugins-root" }}
```

```suggestion
      if: ${{ matrix.composeConfig.name == 'dynamic-plugins-root' }}
```

Looks like this makes the workflow invalid: https://github.com/redhat-developer/rhdh-local/actions/runs/15417339730?pr=48